src/components/page-layout/components/layouts/example/appropriate-image.js

```diff
@@ -1,12 +0,0 @@
-import { scopeAppropriateImage } from '@mapbox/appropriate-images-react';
```

Comment:
Note: I'm not sure why this file is here, it appears unused, so let's get rid of it.